clients/drcachesim/tracer/tracer.cpp

```diff
@@ -898,6 +898,37 @@ insert_filter_addr(void *drcontext, instrlist_t *ilist, instr_t *where, user_dat
     return reg_idx;
 }
 
+#ifdef AARCH64
+static bool
+is_dc_zva_instr(instr_t *instr)
+{
+    return instr_get_opcode(instr) == OP_sys &&
```

Comment:
I would expect a XXX or TODO i# comment about this being non-ideal and that we want a proper opcode (I guess the IR back-aligning supercedes this...even so...)